package.json

```diff
@@ -13,17 +13,16 @@
     }
   },
   "lint-staged": {
-    "*.js": [
-      "eslint --fix",
-      "git add"
+    "src/**/*.ts": [
+      "prettier --write"
     ],
+    "*.js": "eslint --cache --fix",
     "*.ts": [
-      "tslint --fix",
-      "git add"
-    ]
+      "tslint --fix"
```

Comment:
shouldn't we preserve `git add`? otherwise re-formatted file need to be added manually.

Reply:
I think the latest lint-staged does this at the end by default

Reply:
IIRC, this depends upon the version of `lint-staged` with recent versions not requiring it and warning if it's present. The version in the repo looks pretty old, though.

Reply:
Yes, either we bump up pkg or need it back.

Reply:
@kwonoj I added a comment where I updated lint-staged. The git add is no longer required.